brotli.go

```diff
@@ -0,0 +1,15 @@
+package httpgzip
+
+import (
+	"io"
+)
+
+func (fs *fileServer) findBrotliFile(fpath string) io.ReadSeeker {
```

Comment:
Totally optional suggestion: consider calling this `maybeFindBrotliFile`. I find that to be a nice naming pattern for things like this.